src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -282,4 +292,36 @@ export default App.extend({
 
     this.showChildView('tooltip', tooltipView);
   },
+  showEditActionsSidebar() {
+    const headerView = new BulkEditActionsHeaderView({
+      actions: this.getState().getSelected(this.collection),
+    });
+
+    const modal = Radio.request('modal', 'show:sidebar', {
+      headerView: headerView,
+    });
+
+    this.listenTo(headerView, {
+      'delete'(actions) {
```

Comment:
I think we're going to need to do something like `actions.destroy();`  that function on the collection will need to do the batch delete and then on success... it'll probably have to simulate `model` `destroy` events so that any collection or whatever that's displaying that model anywhere handles it.  
_probably_ what we need to do is disable the delete button on click and not `destroy` the modal or clear the state until the patch delete succeeds. (which will remove the models from the collection without from the destroy event)

Reply:
I just made this method into a separate card I'm going to look at real quick like.